types/highlight.js/index.d.ts

```diff
@@ -158,6 +158,13 @@ declare namespace hljs
 	}
 }
 
+declare module 'highlight.js/lib/highlight.js' {
```

Comment:
Could you add a test for these module exports?

Something like this would suffice:
https://github.com/rtfpessoa/diff2html/pull/279/files#diff-b343dabfda22e9aaae3240a5add32efb

Reply:
@nikeee As I tried to explain in the main text. I cannot get the import of those modules in here to work. It fails saying they were not found. While in diff2html it worked fine.
Any suggestions?

Reply:
When using your new type defintions on a clean project, only the first one wroks:

```
import * as c from "highlight.js";
import * as b from "highlight.js/lib/highlight.js";
import a = require("highlight.js/lib/highlight.js");
```

Reply:
Then, how can I add tests for those new modules?